src/components/FileInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from "react";
+import { textSans } from "@guardian/src-foundations/typography";
+import { css } from "@emotion/core";
+import { neutral } from "@guardian/src-foundations";
+
+interface FileInputProps {
+    required: boolean;
+    name: string;
+    label: string;
+}
+
+const optionalLabelStyles = css`
+    ${textSans.small({ fontStyle: "italic" })};
+    color: ${neutral[46]};
+`;
+
+const labelStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })};
+`;
+
+const optionalLabel = (required: boolean): ReactElement | null =>
+    required ? null : <span css={optionalLabelStyles}>Optional</span>
+
+const FileInput = (props: FileInputProps): ReactElement =>
+    <>
+        <label css={labelStyles} htmlFor={props.name}>{props.label} {optionalLabel}</label>
```

Comment:
Isn't `optionalLabel` a function? Do you need to call it with a value for `required`?